app/form_models/publishers/job_listing/contract_information_form.rb

```diff
@@ -0,0 +1,40 @@
+class Publishers::JobListing::ContractInformationForm < Publishers::JobListing::VacancyForm
+  include ActiveModel::Attributes
+  attr_accessor :contract_type, :fixed_term_contract_duration, :is_parental_leave_cover, :working_patterns, :working_patterns_details, :is_job_share
+
+  validates :contract_type, inclusion: { in: Vacancy.contract_types.keys }
+  validates :fixed_term_contract_duration, presence: true, if: -> { contract_type == "fixed_term" }
+  validates :is_parental_leave_cover, inclusion: { in: ["true", "false", true, false] }, if: -> { contract_type == "fixed_term" }
```

Comment:
Think these 2 (parental leave cover and job_share) would be better modelled as boolean attributes, so that we avoid all the confusion between string and boolean values

Reply:
The change @starswan refers to is here: https://github.com/DFE-Digital/teaching-vacancies/pull/7575/files#diff-29262df3ad4897716936c1619c64fff94b3b6e020089d67164f155bb691db04c